src/SDDPoptimize.jl

```diff
@@ -169,81 +335,41 @@ end
 
 
 """
-Make a forward pass of the algorithm
-
-Simulate a scenario of noise and compute an optimal trajectory on this
-scenario according to the current value functions.
+Compute value of Bellman function at point xt. Return V_t(xt)
 
 Parameters:
-- model (SPmodel)
-    the stochastic problem we want to optimize
+- model (SPModel)
+    Parametrization of the problem
 
 - param (SDDPparameters)
-    the parameters of the SDDP algorithm
-
-- display (Bool) - Default is false
-    If specified, display progression in terminal
-
-
-Returns :
-- V (Array{PolyhedralFunction})
-    the collection of approximation of the bellman functions
-
-- problems (Array{JuMP.Model})
-    the collection of linear problems used to approximate
-    each value function
-
-"""
-function optimize(model::SPModel,
-                  param::SDDPparameters,
-                  display=true::Bool)
-
-    # Initialize value functions:
-    V, problems = initialize_value_functions(model, param)
-
-    if display
-      println("Initialize cuts")
-    end
-
+    Parameters of SDDP
 
-    stopping_test::Bool = false
-    iteration_count::Int64 = 0
-    n::Int64 = param.forwardPassNumber
+- t (Int64)
+    Time t where to solve bellman value
 
+- Vt (Polyhedral function)
```

Comment:
For example, for the naming, is it a good idea to call the approximation of Value function by Vt ?